lib/database/bootstrapper_runtime/Dockerfile

```diff
@@ -1,13 +1,11 @@
 ARG PYTHON_VERSION
 FROM --platform=linux/amd64 public.ecr.aws/lambda/python:${PYTHON_VERSION}
 
-ARG PGSTAC_VERSION
-RUN echo "PGSTAC_VERSION: ${PGSTAC_VERSION}"
 RUN echo "PYTHON_VERSION: ${PYTHON_VERSION}"
 
 WORKDIR /tmp
 
-RUN pip install httpx psycopg[binary,pool] pypgstac==${PGSTAC_VERSION} -t /asset
+RUN pip install httpx psycopg[binary,pool] pypgstac -t /asset
```

Comment:
we don't really need to set the pypgstac version here. In theory pypstac 0.9.2 should be able to run the migration step for 0.8

Reply:
good idea - so as long as we install the latest version the bootstrapper should be able to do any migration.